src/transformer/transformer.ts

```diff
@@ -26,31 +26,33 @@ const transformer:
 export { transformer };
 
 function visitNode(node: ts.CallExpression, declaration: ts.FunctionDeclaration): ts.Node {
-  const nodeToMock: ts.TypeNode = node.typeArguments[0];
+  const nodeToMock: ts.TypeNode | undefined = node.typeArguments?.[0];
```

Comment:
Considering that this visitNode already passed the check of typeArguments, I would create an interface that extends ts.CallExpression that always as typeArguments. Is that possible?

Reply:
I guess we can adjust the signature to something like `ts.CallExpression & { typeArguments: NonNullable<ts.CallExpression['typeArguments']> }`

Reply:
I think we could use (node: ts.CallExpression & { typeArguments: NodeArray<TypeNode> })